rsts/deployment/sandbox.rst

```diff
@@ -186,32 +102,39 @@ We'll proceed like with :ref:`locally hosted flyte <deploy-sandbox-local>` with
 Flyte configuration on your remote cluster.
 
 
-#. The Flyte sandbox can be deployed via a helm chart. From the root dir of the flyte repo run ::
+#. Add Helm repo for flyte ::
 
-    helm repo add flyte https://flyteorg.github.io/flyte
-    helm install -n flyte -f values.yaml --create-namespace flyte flyte/flyte
+    helm repo add flyteorg https://helm.flyte.org
 
-#. For customizations instructions, see ``/charts/flyte/README.md`` in the flyte repo.
+#. Install Flyte dependency helm chart (this will install the minio, Postgres, Kubernetes-dashboard, and contour) ::
 
-#. You can now port-forward (or if you have load-balancer enabled then get an LB) to connect to remote FlyteConsole, as follows ::
+    helm install -n flyte flyte-deps flyteorg/flyte-deps --create-namespace --set webhook.enabled=false,minio.service.type=LoadBalancer,contour.enabled=true,contour.envoy.service.type=LoadBalancer,kubernetes-dashboard.service.type=LoadBalancer
 
-    kubectl port-forward --address 0.0.0.0 svc/flyte-contour-envoy 30081:80 -n flyte
+#. Install flyte-core chart ::
 
-#. Open the console http://localhost:30081/console.
+    helm install flyte flyteorg/flyte-core -n flyte -f https://raw.githubusercontent.com/flyteorg/flyte/master/charts/flyte-core/values-sandbox.yaml --wait
+
+#. Verify Flyte deployment ::
```

Comment:
We should say what to do to verify... make sure all pods are "Running"... if you see anything that's crashing, check them in this order: `postgres, minio, flyteadmin, datacatalog, flytepropeller....` etc?